app/helpers/cms_helper.rb

```diff
@@ -167,7 +167,8 @@ def get_resource_links
   def linkify(url, fragment_link)
     # If it is a file as opposed to a link
     if fragment_link =~ /(file)/ 
-      file = @cms_page.fragments.find { |fragment| fragment.tag == 'file' }
+      file = @cms_page.fragments.find_by(tag: 'file')
+      return '' if file.attachments.first.blank?
```

Comment:
Still receiving an issue with this on certain pages, like http://localhost:3000/resources/august-2019-update-of-the-wdpa - specifically a no method error as `file` is nil. I did have a closer look at this, and I found that even though the `fragment_link` was `:resource_file`, the tag was `text`, not `file`! However, this may be an unrelated issue that is most likely caused by the seed import.